content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-kubeconfig.md

```diff
@@ -0,0 +1,24 @@
+---
+title: kubeadm kubeconfig
+content_type: conceito
+weight: 90
+---
+
+`kubeadm kubeconfig` fornece utilitários para gerenciar arquivos kubeconfig.
+
+Exemplos de como usar `kubeadm kubeconfig user`, consulte [Gerando arquivos kubeconfig para usuários adicionais](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs#kubeconfig-additional-users).
+
+
+## kubeadm kubeconfig {#cmd-kubeconfig}
+
+{{< tabs name="tab-kubeconfig" >}}
+{{< tab name="visão geral" include="generated/kubeadm_kubeconfig.md" />}}
+{{< /tabs >}}
+
+## kubeadm kubeconfig user {#cmd-kubeconfig-user}
+
+Esse comando pode ser usado para gerar um arquivo kubeconfig para um usuário adicional.
+
+{{< tabs name="tab-kubeconfig-user" >}}
+{{< tab name="usuário" include="generated/kubeadm_kubeconfig_user.md" />}}
```

Comment:
This tab title was translated, but it refers to an actual subcommand. Let's keep it as "user" instead.